### PR TITLE
Updated k8s_ha documentation for redis as backend coordination

### PR DESCRIPTION
**Pull request by arms11**
Related to https://github.com/StackStorm/stackstorm-ha/pull/169/

### CODE REVIEW

**Comment by arms11** [comment]
@armab - Please review. Thanks.

File: docs/source/install/k8s_ha.rst [hunk 1]
```diff
@@ -301,10 +301,10 @@ By default ``3`` nodes of RabbitMQ are deployed via K8s StatefulSet.
 For more advanced RabbitMQ configuration, please refer to official `rabbitmq-ha <https://github.com/helm/charts/tree/master/stable/rabbitmq-ha>`_
 Helm chart repository, - all settings could be overridden via ``values.yaml``.
 
-etcd
+redis
 ____
```
Comment (armab) [inline docs/source/install/k8s_ha.rst:305]: Check the CI failure here:  https://app.circleci.com/pipelines/github/StackStorm/st2docs/278/workflows/623cf438-4d10-406d-9a80-2e8520151f69/jobs/2912

```
Warning, treated as error:
/home/circleci/project/docs/source/install/k8s_ha.rst:305:Title underline too short.
```

**Review by armab — CHANGES_REQUESTED** [review]
Thanks for updating the st2docs!

A few changes:
- CI is not happy with the syntax. Provided a pointer in the comment above.

- Similar to previous Readme.md changes from your PR: https://github.com/arms11/stackstorm-ha/commit/15abe2fa73c1adc1474b422f11b98ab21dcbd3f0 there are a few more places where we can replace `etcd` with `redis`.

**Comment by arms11** [comment]
Thanks @armab . Duh...I did not realize there were other references. I replaced them as appropriate. Checked the build also have passed. Please review!

**Review by armab — APPROVED** [review]
Looks good!

Thanks for syncing the Documentation, following the https://github.com/StackStorm/stackstorm-ha/pull/169 change :+1:

**Pull request merged**